### DISCUSSION TITLE
[New Feature] DockingOffset supports Height

### DISCUSSION THREAD

**Discussion by deathreaperz** [Phobos-developers/Phobos#945] [Ideas and feedback]
Wondering if Helipad DockingOffset supports height, then it will have various structures design + the aircraft won't take time to limbo.
```
[HELIPAD ART]
DockingHeight=(int/value) ;def 0
```

**Comment by TAK02** [comment]
IIRC, the Offset takes and parses height properly; the problem is the Aircraft will "jump" up and down between the Dock in mid-air and the ground beneath it.
What you'd be asking for is a fix for a vanilla bug, I think.

**Comment by McPwny** [comment]
can confirm. dockingoffset is forward, lateral, height format already. its just that non 0 inputs on the height cause planes to not land so it is a vanilla bug. i think it was patched in the chinese ares, too.
